examples/ArduinoIoTCloud_Travis_CI/arduino_secrets.h

```diff
@@ -1,9 +1,9 @@
 /* (MKR1000, MKR WiFi 1010) */
-#define SECRET_SSID "YOUR_WIFI_NETWORK_NAME"
-#define SECRET_PASS "YOUR_WIFI_PASSWORD"
+#define WIFI_SECRET_SSID "YOUR_WIFI_NETWORK_NAME"
```

Comment:
This is not going to work, the parser looks for SECRET string as being the first part of the word.
Then please rename SECRET_WIFI_SSID and so on.

Reply:
Done :smile: